docs/content/contribute/styleguide.md

```diff
@@ -0,0 +1,324 @@
+
+Where citations are given, we permit any consistent and legible citation format that includes the vital information needed, typically a work's title, its authorship or creator, publication date, editor if applicable, and available links and time stamps.
+
+See also [Reference Format for NIST Publications](https://inet.nist.gov/library/publishing-support-nist-publications/reference-format-nist-publications) (requires NIST login).
+
+### Headers and heading levels
+
+This rule may not apply if your page is short, you do not need tables of contents showing correctly partitioned and nested sections, or you do not care.
+
+In Markdown, section headers are indicated using '#' characters, the count of such characters indicating heading level such that `###` indicates `h3` (third-level header). The level of header is taken as an indicator of structural demarcations (sections or `div` elements) in the presentation, such that a third-level section is considered inside (not next to) the second level section that (immediately) precedes it.
+
+It is up to you to see to it that your headers indicate properly nested sections:
+
+- `h1` may be directly followed by `h1` or `h2`
+- `h2` may be directly followed by `h3`, `h2`, or `h1`
+- `h3` may be directly followed by `h4`, `h3`, `h2` or `h1`
+- etc.: never go down more than one level at a time
+
+This is to ensure that tables of contents written in the web site build are consistent and legible. Previewing in the site is a good way to check the nesting.
+
+An example of a properly nested sequence is  `h1 h2 h2 h3 h3 h4 h2 h1`:
+
+- `h1` section
+  - `h2` subsection
+  - `h2` subsection
+    - `h3` 3rd level section
+    - `h3`
+      - `h4` 4th level 
+  - `h2` new subsection
+- `h1` new section
+
+But the sequence `h1 h3 h4` is not nested neatly because a third-level section never appears directly inside a first-level section.
+
+(Exercise: write a validator that can report when this rule is broken, over Markdown or HTML inputs.)
+
+### Images
+
+If making a diagram or schematic image, see above under [Diagrams](#diagrams).
+
+Images should always be provided with `alt` values providing short descriptions for navigation devices and screen readers, enclosed in `[ ]` (brackets) in the notation. Using Markdown:
+
+```
+![OSCAL gears icon](../oscal-gears-icon-32.png)
+```
+
+For an image ![OSCAL gears icon](../oscal-gears-icon-32.png).
+
+### Inline code
+
+Markdown supports code snips inline with the back-tick notation. Code almost always appears in monospace font, sometimes with further visual emphasis. References to things can also be code -- for example, it may be useful to use code `h1` (XPath notation for 'An h1 element') or `\<h1>` (An HTML or OSCAL open tag for the same thing) to refer unambiguously to elements named <q>h1</q>. Only do so consistently and in a way that communicates the idea to the prospective user or reader. If using XPath, for example, state as much.
```

Comment:
Minor question: did we want to use emdashes here in the place of `--`?

Reply:
Probably. Shall I throw in `&mdash;` and see what happens? (Should edit those out though as dashes are bad.)